packages/trader/src/Modules/Contract/Containers/__tests__/contract-replay-widget.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { ChartBottomWidgets, ChartTopWidgets, DigitsWidget, InfoBoxWidget } from '../contract-replay-widget';
+
+jest.mock('../contract-replay-widget.tsx', () => ({
+    ...jest.requireActual('../contract-replay-widget.tsx'),
+    DigitsWidget: () => <div>MockedDigitsWidget</div>,
```

Comment:
Okay, I'm not sure, but in existing test files (not only in trader package) `jest.fn` is used, e.g:
```
jest.mock('@deriv/shared', () => ({
    ...jest.requireActual('@deriv/shared'),
    isDesktop: jest.fn(() => true),
    formatMoney: jest.fn(),
}));
```

Reply:
yeah jest.fn() works for functions. This is a component thats why it's done the way it is